main.go

```diff
@@ -217,33 +229,56 @@ func logSavedReview(savedReview *SavedReview) {
 	log.Println("-------")
 }
 
-func saveReviewToFile(filePath, review string, reviewComments []*github.DraftReviewComment, action string) error {
+func saveReviewToFile(reviewFilePath, review string, reviewComments []*github.DraftReviewComment, action string) error {
```

Comment:
Consider changing the function name 'saveReviewToFile' to accurately reflect the new file handling approach.